src/components/select/filterable-select/filterable-select.component.js

```diff
@@ -388,7 +388,10 @@ const FilterableSelect = React.forwardRef(
     function handleTextboxMouseDown(event) {
       isMouseDownReported.current = true;
 
-      if (event.target.attributes["data-element"].value === "input") {
+      if (
+        event.target.attributes["data-element"] &&
+        event.target.attributes["data-element"].value === "input"
```

Comment:
FYI, this could be `event.target.attributes["data-element"]?.value`

Reply:
Ah good point for some reason I thought optional chaining didn't work